### PR TITLE
📝 Improve contributor documentation

### PR DESCRIPTION
**Pull request by tzvipm**
Closes https://github.com/Shopify/quilt/issues/242

### CODE REVIEW

File: .github/CONTRIBUTING.md [hunk 1]
```diff
@@ -1,66 +1,87 @@
...
+
+**Note** This project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
+
+4. Log in to [Shipit](https://shipit.shopify.io/shopify/quilt/production)
 
-We're working on improving the release experience for new contributors. In the meantime, please ping one of the `quilt` repo owners when you're ready to merge a new PR into `master`, and we will help you with orchestrating a new release via Shipit.
+5. When CI is 🍏 on the tag commit, press `Deploy` to update packages on npm.
 
-This project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
+## External Contributors
```
Comment (kaelig) [inline .github/CONTRIBUTING.md]: That's something @BPScott may be able to help with (in regards to the CLA signing process and such).

File: README.md [hunk 2]
```diff
@@ -1,19 +1,24 @@
...
-
 [![License: MIT](https://img.shields.io/badge/License-MIT-green.svg)](LICENSE.md)
 [![Build Status](https://travis-ci.org/Shopify/quilt.svg?branch=master)](https://travis-ci.org/Shopify/quilt)
 [![codecov](https://codecov.io/gh/Shopify/quilt/branch/master/graph/badge.svg)](https://codecov.io/gh/Shopify/quilt)
 [![Greenkeeper badge](https://badges.greenkeeper.io/Shopify/quilt.svg)](https://greenkeeper.io/)
+[![lerna](https://img.shields.io/badge/maintained%20with-lerna-cc00ff.svg)](https://lernajs.io/)
+
+A loosely related set of packages for JavaScript/Typescript projects at Shopify. These libraries compose together to help you create performant modern JS apps that you love to develop and test. These packages are developed primarily to be used on top of the stack we like best for our JS apps; Typescript for the flavor, Koa for the server, React for UI, Apollo for data fetching, and Jest for tests. That said, you can mix and match as you like.
```
Comment (kaelig) [inline README.md]: /nit TypeScript (with an uppercase S)

File: .github/CONTRIBUTING.md [hunk 3]
```diff
@@ -1,66 +1,87 @@
...
 
-## Testing
+### Testing
 
 The packages in Quilt are used in mission-critical production scenarios. As such, we try not to merge any untested code. The coverage doesn't strictly need to be 100% across the board, but testing should remain a primary concern.
 
-## TODO Comments
+### TODO Comments
 
 TODO comments may seem like a great placeholder for work in progress. We prefer top handle this in a different way, using a combination of feature branches and github issues.
```
Comment (TheMallen) [inline .github/CONTRIBUTING.md]: > prefer top

prefer too*

File: .github/CONTRIBUTING.md [hunk 4]
```diff
@@ -1,66 +1,87 @@
...
 
 If your changes are complete in functionality, but you're not quite happy with auxillary things like documentation or testing, then feel free to make a github issue to track the work that needs to be done. These issues should be linked in the PRs that need a bit more work. This will allow context to be drawn from the code in a more trackable way than a TODO comment. Also, it allows the PR reviewers to see that the documentation or testing is purposefully incomplete and that an appropriate issue exists to track the follow-up work.
 
-### Feature branches
+#### Feature branches
 
 Another option, if you'd like to break work down into reviewable chunks, is to use a feature branch. This would be an initially empty branch that contains the entirety of your feature. Additional units of work can be distributed across several PRs into the feature branch, merged independently, and then the feature branch can be merged as a complete unit into the master branch, when it's ready.
 
-## Releasing a new version
+## Releasing
```
Comment (TheMallen) [inline .github/CONTRIBUTING.md]: We might want to mention that the contributor should not alter any version numbers in package.jsons

File: .github/CONTRIBUTING.md [hunk 5]
```diff
@@ -1,66 +1,87 @@
...
+The release process currently invovles some manual steps to complete. Please ping one of the repo owners in the `#web-foundations` slack channel when you're ready to merge a new PR into `master`, and we will orchestrate a new release.
+
+### For repo owners
+
+**Note** these steps require admin access to the `Shopify/quilt` github repo.
+
+1. Ensure you have the latest `master` branch including all tags:
+
+```
+git checkout master && git pull origin master
```
Comment (TheMallen) [inline .github/CONTRIBUTING.md]: `--tags`

File: .github/CONTRIBUTING.md [hunk 6]
```diff
@@ -1,66 +1,87 @@
...
+
+2. Begin the release process:
+
+```
+yarn release
+```
+
+3. Follow the prompts to choose a version for each package.
+
+**Note** This project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
```
Comment (TheMallen) [inline .github/CONTRIBUTING.md]: Probably better stated as `These packages adhere...`

File: README.md [hunk 7]
```diff
@@ -35,9 +40,13 @@ This repository uses [lerna](https://github.com/lerna/lerna) to manage it's pack
...
 
-## Contribute
+## Want to contribute?
+
+Check out our [Contributing Guide](./.github/CONTRIBUTING.md)
+
+## Questions?
 
-Check out our [Contributing Guidelines](CONTRIBUTING.md)
+For Shopifolk, you can reach out to us in Slack on the `#web-foundation` channel. For external inquiries, we welcome bug reports, enhancements, and feature requests via Github issues.
```
Comment (TheMallen) [inline README.md]: We should still have the links to the guidelines here

Comment (tzvipm) [reply README.md]: We do, just above in the Contributing section. This section is for other questions.

**Pull request merged**